airflow/serialization/base_serialization.py

```diff
@@ -22,17 +22,18 @@
 import datetime
 import enum
 import logging
-from typing import TYPE_CHECKING, Dict, Optional, Union
+from inspect import Parameter
+from typing import Dict, Optional, Set, Union
 
 import pendulum
 from dateutil import relativedelta
 
-import airflow
+from airflow import DAG
```

Comment:
Are we not using `from airflow.models import DAG` to remove cyclic import? 

Now we would have 2 version in the core as well. `from airflow import DAG` and `from airflow.models import DAG` in the core codebase

Reply:
The change to standardize DAG usage is coming :). I changed it from airflow.models import DAG here. The previous change was not actually working as I wanted. As @ashb pointed out, `from airflow.models import DAG' also runs __init__.py of 'airflow' package, so it would not actually change anything.